tests/e2e/framework/common.go

```diff
@@ -1451,3 +1471,28 @@ func (f *Framework) AssertHasCheck(suiteName, scanName string, check compv1alpha
 
 	return nil
 }
+
+func (f *Framework) ReRunScan(scanName, namespace string) error {
```

Comment:
This will conflict with a couple other patches I have that need this. Just a heads up for the rebase.